src/button/index.js

```diff
@@ -14,15 +14,16 @@ const styles = {
     textTransform: 'uppercase',
     background:'rgba(0,0,0,0)',
     fontSize: 14,
-    padding: [14, 54],
+    padding: [18, 54],
```

Comment:
Unfortunately the values for top and bottom padding in the styleguide seem to be wrong. Your change to letter spacing down below is good though, you can keep that!